packages/holocron-module-register-webpack-plugin/__tests__/HolocronModuleRegisterPlugin.spec.js

```diff
@@ -130,6 +134,6 @@ describe('HolocronModuleRegisterPlugin', () => {
 
     await waitForWebpack(options);
     const fileContents = fs.readFileSync(path.join(buildPath, outputFileName)).toString();
-    expect(fileContents.endsWith(`Holocron.registerModule("${moduleName}", ${holocronModuleName});})();`)).toBe(true);
+    expect(fileContents).toContain('Holocron.registerModule("some-module", holocronModule-some-module);');
```

Comment:
The kebab-case `holocron-some-module` can't be right. Isn't that supposed to be a variable name?

Reply:
Yes, however this tests shows that the plugin takes the name specified. We are specifying a non-camel-case module variable name up on [Line 122](https://github.com/americanexpress/holocron/pull/154/files/147873fd20de2dad6c11f6a5084bb63bfa8b1dd7#diff-3da1ec6afcf453657aa41c4ca855d04a8c29fcbe05bc0063f376598fc29d4b5aR122)

I have not changed this, just made the 'snapshot' literal instead of templated, so we can visually see that the name passed is accepted and inserted.

Reply:
Addressed in ea09da2